clusters/app.ci/rhcos-redirector/apici_ingresscontroller.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  name: tlsv11
+  namespace: coreos
+spec:
+  domain: releases-art-rhcos.svc.ci.openshift.org
```

Comment:
@Miciah Does it impact the route generated by the same ingress with the hostname `rhcos-redirector.ci.openshift.org`?

Reply:
Will this IngressController impact on routes from other namespaces?

Reply:
> @Miciah Does it impact the route generated by the same ingress with the hostname `rhcos-redirector.ci.openshift.org`?

No, it doesn't affect the generation of the route.  The ingresscontroller's `spec.domain` affects the wildcard DNS record that gets published and the name that gets published in the route's status.

Reply:
> Will this IngressController impact on routes from other namespaces?

Not unless they're labeled `type: tlsv11`.

Reply:
/hold

Looking for a better option than wildcard DNS as spec.domain.